### PR TITLE
Replace Cytoscape with ReactForceGraph

### PR DESCRIPTION
**Pull request by trmiller**
Simplify the graph UI, replace Cytoscape with ReactForceGraph, remove unnecessary filters

### CODE REVIEW

File: apollo/client.ts [hunk 1]
```diff
@@ -1,41 +1,23 @@
...
-  },
-}
 function createApolloClient() {
   return new ApolloClient({
     ssrMode: typeof window === 'undefined',
     // link: createIsomorphLink(),
     uri: "http://localhost:3000/api/graphql",
-    defaultOptions: defaultOptions,
     cache: new InMemoryCache(),
+    defaultOptions: {
```
Comment (mlieberman85) [inline apollo/client.ts:13]: Can you just add a TODO here to fix up the cache once everything else is stabilized?

File: components/guac/packageVersionSelect.tsx [hunk 2]
```diff
@@ -32,11 +34,40 @@ const PackageVersionSelect = ({ label, options, setPackageVersionFunc, setGraphD
     });
     let q = packageNamespacesQuery.then(
       res => {
-        setGraphDataFunc(res.data.packages);
+        const graphData = {nodes:[], links:[]}
+        const parsedNode = ParseNode(res.data.packages[0]) // is this a problem?
```
Comment (mlieberman85) [inline components/guac/packageVersionSelect.tsx:38]: What does this comment mean?

File: components/guac/toggleSwitch.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    return (
+        <label>
+            <strong>{label}</strong>
+            <input type="checkbox" defaultChecked={isToggled} onClick={callback} />
+            <span className="toggle_span"/>
+        </label>
+    )
+}
```
Comment (mlieberman85) [inline components/guac/toggleSwitch.tsx:18]: nit: no new line 🙃

File: pages/index.tsx [hunk 4]
```diff
@@ -27,18 +28,28 @@ export default function Home() {
   const [packageNamespaces, setPackageNamespaces] = useState(initialPackageNamespaces)
   const [packageNames, setPackageNames] = useState(initialPackageNamespaces)
   const [packageVersions, setPackageVersions] = useState(initialPackageNamespaces)
+  const [renderedInitialGraph, setRenderedInitialGraph] = useState(false)
+
+  // query params - this is used to pass in node id's to go get
+  // test with nodeId: 3597773
```
Comment (mlieberman85) [inline pages/index.tsx:34]: This nodeid is only valid during your import of data. If you rerun ingestion it should provide a different set of node ids

Comment (trmiller) [reply pages/index.tsx:34]: yeah I just forgot to delete this.  fixing

File: pages/index.tsx [hunk 5]
```diff
@@ -150,21 +142,109 @@ export default function Home() {
...
-        {
-          cjsGraphData.map((d)=> {
-            const [start, graphData] = processDataForCytoscape(d.data);
-            return <Graph key={d.key} layout="dagre" writeDetails={()=>{}} startNode={start} graphData={graphData} /> 
-          })
-        }
-      </div>
+      <div className="grid grid-cols-3">
+        <div className="w-full items-left justify-left font-mono text-sm p-24 lg:col-span-1">
+          {/* <h2>Highlight Nodes</h2>
```
Comment (mlieberman85) [inline pages/index.tsx:147]: Why commented out? Do you still plan to work on it? If so can you just add a todo?

File: styles/globals.css [hunk 6]
```diff
@@ -25,3 +25,58 @@ body {
...
+}
+
+strong {
+  position: absolute;
+  left: 100%;
+  width: max-content;
+  line-height: 30px;
+  margin-left: 10px;
+  cursor: pointer;
+}
```
Comment (mlieberman85) [inline styles/globals.css:82]: newline

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Pull request merged**